src/main/java/camp/nextstep/edu/rebellion/racing/SnapShotEntry.java

```diff
@@ -0,0 +1,31 @@
+package camp.nextstep.edu.rebellion.racing;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class SnapShotEntry {
+    private final List<RacingCar> cars;
+
+    public SnapShotEntry(List<RacingCar> cars) {
+        this.cars = cars;
+    }
+
+    public List<RacingCar> getCars() {
+        return Collections.unmodifiableList(cars);
+    }
+
+    public List<String> getWinners() {
+        return cars.stream()
+                .filter(car -> car.getPosition() >= getMostDistancePosition())
+                .map(car -> car.getName())
+                .collect(Collectors.toList());
```

Comment:
우승자를 구할때 최대 거리는 한번만 구해도 되지 않을까 하는데 어떻게 생각하시나요 :) ?
```suggestion
        int mostDistancePosition = getMostDistancePosition();
        return cars.stream()
                .filter(car -> car.getPosition() >= mostDistancePosition)
                .map(car -> car.getName())
                .collect(Collectors.toList());
```

Reply:
아 맞네요 뭔가 람다의 멋에 심취(?) 하다 보니 이런 기초적인 실수를 😢 
다음 PR에 수정 하겠습니다 
꼼꼼히 봐주셔서 감사합니다 🙇